R/dots.R

```diff
@@ -97,7 +97,8 @@ dots_clean_empty <- function(dots, is_empty, ignore_empty) {
 
   if (n_dots) {
     which <- match.arg(ignore_empty, c("trailing", "none", "all"))
-    switch(which,
+    switch(
+      which,
```

Comment:
Thanks @lionel- for chiming in. I like the idea that anything before the triple dots *may* stay after the opening paren, we should be able to cover most cases (apart from `ifelse()`). Unfortunately, doing this accurately also seems to require knowledge about the callee's argument list, which we don't have yet.

Let's simply avoid adding a line break for `strict = FALSE` .